### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by aatishpandey**
image
Includes:
Style for navigation.

### CODE REVIEW

File: components/component-stylesheets/nav.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.nav{
+    display: flex;
+    width: 100%;
+    column-gap: 50px;
```
Comment (shubhamkumar648) [inline components/component-stylesheets/nav.css]: you can use rem unit for better resonsive

**Review by shubhamkumar648 — COMMENTED** [review]
Remaining looks good

**Comment by adithyark17** [comment]
looks great! good to go

File: components/component-stylesheets/nav.css (hunk 1 above)
Comment (aatishpandey) [reply components/component-stylesheets/nav.css]: Thanks for pointing out, fixed in recent commit

**Pull request merged**